projects/soft_999_shicheng_lu_fw_102/src/main.c

```diff
@@ -0,0 +1,39 @@
+#include <stdint.h>
+
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define DELAY_MS 500
+
+typedef struct Counters {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+
+void increment_counters(SoftTimerId id, void *context) {
+  Counters *counter = context;
```

Comment:
I'm not a big fan of naming it `counter` since it technically holds counter a and counter b, maybe we could give this a better name?

`counter` sort of implies its just a single counter